drools-wb-services/drools-wb-verifier/drools-wb-verifier-webworker/src/main/resources/org/drools/workbench/services/verifier/plugin/backend/test.txt

```diff
@@ -0,0 +1 @@
+hello there
```

Comment:
This seems to me as test resource, is it really needed in main resources?

Reply:
Yep, this is in the wrong location.